modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -3624,7 +3678,7 @@ If the number of scheduled tick moves drops by this ratio, a new tick is schedul
 
 === partition_autobalancing_topic_aware
 
-Enable topic-aware partition balancing, which balances a topic's partition replica count on each broker before balancing the cluster's overall partition count. This is helpful because different topics in a cluster can have vastly different load profiles.
+If `true`, Redpanda will prioritize balancing a topic-wise number of partitions on each node, as opposed to balancing the total number of partitions. This should provide better balancing results if topics with diverse partition sizes and load profiles are present in the cluster.
```

Comment:
```suggestion
If `true`, Redpanda prioritizes balancing a topic’s partition replica count evenly across all brokers while it’s balancing the cluster’s overall partition count. Because different topics in a cluster can have vastly different load profiles, this better distributes the workload of the most heavily-used topics evenly across brokers.
```